lib/challenge.js

```diff
@@ -33,7 +64,10 @@
 var pet_owner;
 
 // Answer Starts Here
-
+var pet_owner = {
+  name: 'Chris',
+  address: '241 N Mtn Rd, Dalton MA 01226'
+}
```

Comment:
Awesome. At the end of this like, after the closing curly brace, we are at the end of the statement that begins with `var`. To be explicit, a semicolon following this closing curly brace would show that you intend for this statement to end here. Full credit.